pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/documentation-types.md

```diff
@@ -0,0 +1,274 @@
+---
+$title: Documentation types 
+$order: 1
+description: 'Types of documentaion contributions accepted on amp.dev'
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+author: CrystalOnScript
+---
+Below is a brief outline of the types of documentation contributions accepted on amp.dev. 
+
+
+## Introductory tutorial
+
+Introductory tutorials help the developer understand the general idea of the technology. It starts them off coding and ends with a complete basic “Hello World” project. Introductory tutorials demonstrate how to build a key feature of AMP in a step-by-step process. Pair Introductory tutorials with inline code samples and a downloadable sample, that requires minimal tweaks by the developer to run.
+
+amp.dev examples:
+
+
+
+*   [Create your first AMP page]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/start/create.md', locale=doc.locale).url.path}}?format=websites)
+*   [Create your first AMP story]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/start/visual_story.md', locale=doc.locale).url.path}}?format=stories)
+*   [Create your first AMPHTML ad]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/start/create_amphtml_ad.md', locale=doc.locale).url.path}}?format=ads)
+
+<table>
+  <tr>
+   <td>
+<strong>Do</strong>
+   </td>
+   <td><strong>Don’t</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Provide guidance with brief explanations and minimal steps. 
+   </td>
+   <td>Dive deep into project nuances. There may be many ways to accomplish the tutorial, but we are presenting <em>this</em> way.
```

Comment:
Maybe clarify a little more, like: "There may be many was to accomplish the outcome of the tutorial, but the point isn't to show every route, but a single, good route."